src/pages/HomePage.tsx

```diff
@@ -8,6 +8,7 @@ import { fetchMovies } from "../network/requests";
 
 const HomePage = (): JSX.Element => {
   const [user, setUser] = useState<User | null>(null);
+  // TODO: Provide coherent UX when TMDB is down (when movies == []).
```

Comment:
Hmm can we add this to an issue tracker or something else instead of leaving it as a comment? I think TODOs like this tend to get lost/forgotten.

Reply:
Oh, can we actually remove the todo?